mcpserver/tools.go

```diff
@@ -280,6 +290,10 @@ var EnvironmentUpdateTool = &Tool{
 			mcp.Description("Short description of the work that is happening in this environment."),
 			mcp.Required(),
 		),
+		mcp.WithString("summary",
+			mcp.Description("Summary of all the changes that happened in this environment. This should be formatted like a commit message and always kept up to date `environment_update`."),
+			mcp.Required(),
+		),
```

Comment:
instructions, title, base_image, setup commands, envs, and secrets need to be optional if this is required, and if we're going that far, maybe everything on here should be optional.